addons/l10n_br/data/account.account.template.csv

```diff
@@ -1,766 +1,69 @@
-account_template_3010105010100,3.01.01.05.01.01.00,Variações Cambiais Ativas,"Contas que registram os ganhos apurados em razão de variações ativas Decorrentes da atualização dos direitos de crédito e obrigações, calculados com base nas variações nas taxas de câmbio.
-Atenção:
-1) As variações cambiais ativas decorrentes dos direitos de crédito e de obrigações, em função da taxa de câmbio, são consideradas como receita financeira, inclusive para fins de cálculo do lucro da exploração (Lei nº 9.718, art. 9º c/c art. 17); 
-2) Nas atividades de compra e venda, loteamento, incorporação e construção de imóveis, as variações cambiais ativas são reconhecidas como receita segundo as normas constantes da IN SRF nº 84/79, de 20 de dezembro de 1979, da IN SRF nº 23/83, de 25 de março de 1983, e da IN SRF nº 67/88, de 21 de abril de 1988 (IN SRF nº 25/99, de 25 de fevereiro de 1999).",account.data_account_type_expenses,,l10n_br_account_chart_template
-account_template_3010105010200,3.01.01.05.01.02.00,"Ganhos Auferidos no Mercado de Renda Variável, exceto Day-Trade","Contas que registram:
-a) o somatório dos ganhos auferidos, em cada mês do período de apuração, em operações realizadas nas bolsas de valores, de mercadorias, de futuros e assemelhadas, existentes no País;
-b) os ganhos auferidos nas alienações, fora de bolsa, de ouro, ativo financeiro, e de participações societárias, exceto as alienações de participações societárias permanentes em sociedades coligadas e controladas e de participações societárias que permanecerem no ativo da pessoa jurídica até o término do ano-calendário seguinte ao de suas aquisições; e 
-c) os rendimentos auferidos em operações de swap e no resgate de quota de fundo de investimento cujas carteiras sejam constituídas, no mínimo, por 67% (sessenta e sete por cento) de ações no mercado à vista de bolsa de valores ou entidade assemelhada (Lei nº 9.532, de 1997, art. 28, alterado pela MP nº 1.636, de 1998, art. 2º, e reedições).Considera-se ganho o resultado positivo auferido nas operações citadas acima, realizadas em cada mês, admitida a dedução dos custos e despesas incorridos, necessários à realização das operações.
-Atenção:
-1) Os ganhos auferidos em operações day-trade devem ser informados em conta específica.
-2) O valor correspondente às perdas incorridas no mercado de renda variável, exceto day-trade, deve ser informado em conta específica.
-3) São consideradas as
```

Comment:
This CoA seems to be more complete in terms of account types and probably a lot more correct (I hope), but I wonder if we don't exaggerate by using a simplified chart of accounts.  Imagine you do a demo for an accountant, he will see that everything is possible with this CoA?